code/modules/surgery/organs/subtypes/skrell.dm

```diff
@@ -1,3 +1,51 @@
 /obj/item/organ/internal/liver/skrell
 	alcohol_intensity = 4
 	species = "Skrell"
+
+/obj/item/organ/internal/headpocket
+	name = "headpocket"
+	desc = "Allows Skrell to hide tiny objects within their head tentacles."
+	icon_state = "skrell_headpocket"
+	origin_tech = "biotech=2"
+	w_class = WEIGHT_CLASS_TINY
+	parent_organ = "head"
+	slot = "headpocket"
+	species = "Skrell"
+	actions_types = list(/datum/action/item_action/organ_action/toggle)
+	var/obj/item/weapon/storage/internal/pocket
+
+/obj/item/organ/internal/headpocket/New()
```

Comment:
I tried removing the New()...turns out, can't be done because src and owner aren't defined until the constructor is called :/